src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+	Component,
+	Input,
+	OnInit
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+  <a
+    data-tile="clickable"
+    [ngClass]="{'bx--tile': true, 'bx--tile--clickable': true, 'bx--tile--is-clicked': clicked}"
+    tabindex="0"
+    (click)="handleClick($event)"
+    (keydown)="handleKeyDown($event)"
+    [href]="href"
+    [attr.aria-disabled]="disabled">
+    <ng-content></ng-content>
+  </a>`,
+	styles: ["a[aria-disabled=\"true\"] { opacity: 0.5; pointer-events: none; }"],
+})
+export class ClickableTile implements OnInit {
+
+	static CARRIAGE_RETURN = 13;
+	static SPACE = 13;
+
+	/**
+	 * Sets the `href` attribute on the `ibm-clickable-tile` element.
+	 * @type {string}
+	 * @memberof ClickableTile
+	 */
+	@Input() href = "#";
+
+	/**
+	 * Sets the `aria-disabled` attribute on the `ibm-clickable-tile` element.
+   * Setting this to `true` will disable the clickable tile.
+	 * @type {boolean}
+	 * @memberof ClickableTile
+	 */
+	@Input() disabled = false;
+
+	clicked = false;
+
+	constructor() { }
+
+	ngOnInit() { }
+
+	handleClick(event) {
+		this.clicked = !this.clicked;
+	}
+
+	handleKeyDown(event) {
+		if (event.which === ClickableTile.CARRIAGE_RETURN ||
+			event.which === ClickableTile.SPACE) {
```

Comment:
[Use `event.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) with strings instead ... [`event.which` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which)